drake/CTestCustom.cmake.in

```diff
@@ -0,0 +1,14 @@
+set(CTEST_CUSTOM_COVERAGE_EXCLUDE
+  ".*/test/.*"
+  ".*/thirdParty/.*"
+  ${CTEST_CUSTOM_COVERAGE_EXCLUDE}
+)
+
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_ERRORS 100)
+
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS 100)
+
+set(CTEST_CUSTOM_WARNING_MATCH
+  "(?:(?:((?:[/.]|[A-Za-z]:).+?):(-?\\d+):\\s*([Ww]arning|[Ee]rror)|<.+>:-?\\d+(?::\\s*([Ww]arning|[Ee]rror))?): (.+(?:\\n(?!\\s*(?:[Nn]otice|[Ww]arning|[Ee]rror): )[^/<\\n][^:\\n][^/\\n].+)*)|\\s*([Nn]otice|[Ww]arning|[Ee]rror): (.+))$"
+  ${CTEST_CUSTOM_WARNING_MATCH}
+)
```

Comment:
Is there some implicit reading of this file as we do not explicitly do so in the dashboard script.

Also is the different behavior documented anywhere? It is quite confusing.